packages/loader/container-utils/src/error.ts

```diff
@@ -30,8 +30,9 @@ export class GenericError extends LoggingError implements IGenericError {
     constructor(
         errorMessage: string,
         readonly error: any,
+        props?: ITelemetryProperties,
     ) {
-        super(errorMessage);
```

Comment:
can you add a quick ut here. should be easy given the tests @ksbrar added

Reply:
Yay someone asked for a test!  I just pushed one and was going to rag on everyone for not asking :P